### DISCUSSION TITLE
automation example

### DISCUSSION THREAD

**Discussion by batara666** [servo/servo#27437] [General]
hi awesome servo folks, i'm new in programming and i need servo automation example, my use case for :
access a web
click clack 
disable image &> media load opt

**Comment by jdm** [comment]
@batara666 I recommend looking into the webdriver protocol. Servo has a webdriver server that you can start with the --webdriver=6000 argument.

**Reply by jdm** [reply]
More specifically, we have a wiki page that describes this in more details: https://github.com/servo/servo/wiki/Control-Servo-using-WebDriver
